flytekit/core/context_manager.py

```diff
@@ -349,7 +349,9 @@ def __getattr__(self, item: str) -> _GroupSecrets:
         """
         return self._GroupSecrets(item, self)
 
-    def get(self, group: str, key: Optional[str] = None, group_version: Optional[str] = None) -> str:
+    def get(
+        self, group: str, key: Optional[str] = None, group_version: Optional[str] = None, encode_mode: str = "r"
```

Comment:
what if we make this an enum (containing only 2 options: "r" or "rb")?

Reply:
good idea, I will add in next commit

Reply:
i think using string is ok. It is consistent with what we pass to `with open(fpath, <here>)` . Enum seems a bit overkilled

Reply:
Fair enough. At the very least put a description in the function docstring, @ysysys3074 ?

Reply:
you can do a literal type if you want.

Reply:
@eapolinario added in new commit